docs/en/stack/ml/df-analytics/ml-trained-models.asciidoc

```diff
@@ -13,168 +13,146 @@ information about this process, see <<ml-supervised-workflow>> and
 <<ml-inference>>.
 
 You can also supply trained models that are not created by {dfanalytics-job} but
-adhere to the appropriate 
-https://github.com/elastic/ml-json-schemas[JSON schema]. If you want to use 
-these trained models in the {stack}, you must store them in {es} documents by 
+adhere to the appropriate
+https://github.com/elastic/ml-json-schemas[JSON schema]. If you want to use
+these trained models in the {stack}, you must store them in {es} documents by
 using the {ref}/put-trained-models.html[create trained models API].
 
-In {kib}, you can view and manage your trained models within *{ml-app}* > *Data 
+In {kib}, you can view and manage your trained models within *{ml-app}* > *Data
 Frame Analytics*:
 
 [role="screenshot"]
 image::images/trained-model-management.png["List of trained models in the {ml-app} app in {kib}"]
 
-Alternatively, you can use APIs like 
+Alternatively, you can use APIs like
 {ref}/get-trained-models.html[get trained models] and
 {ref}/delete-trained-models.html[delete trained models].
 
 
 [discrete]
-[[move-between-clusters]]
-== Moving a trained model between clusters
-
-It is a common scenario that the {ml} models are trained in a development or 
-test environment and then used in a production environment. In this case, you 
-need to move your trained model from one cluster to another. The trained model 
-APIs enable you to move your trained model between clusters. The following 
-description shows you the process step by step.
-
-1. (Optional) In the cluster where you trained the model, make the call below by 
-using the console in **Dev Tools** to get the configuration information of your 
-trained models.
+[[export-import]]
+== Exporting and importing models
+
+Models trained in Elasticsearch are portable and can be transferred between
+clusters. This is particularly useful when models are trained in isolation from
+the cluster where they are used for inference. The following instructions show
+how to use https://curl.se/[`curl`] and https://stedolan.github.io/jq/[`jq`] to
+export a model as JSON and import to another cluster.
+
+1. Given a model _name_, find the model _ID_. Using `curl` we can call the
```

Comment:
We try to avoid "we" unless it's a tutorial, which this isn't. Use "you" instead:
```suggestion
1. Given a model _name_, find the model _ID_. You can use `curl` to call the
```